### DISCUSSION TITLE
How to access parameter values for each time-step with callback function policy_param?

### DISCUSSION THREAD

**Discussion by blancamiller** [google/brax#468] [General]
I am using the provided [Brax Training](https://colab.research.google.com/github/google/brax/blob/main/notebooks/training.ipynb) tutorial for PPO + Ant. The notebook provides a user-defined callback function `progress`. Similarly you can specify a `policy_param` function and pass it into `train`. This is where my issues lies. I’m able to get the parameter values for each episode, however, I’d like to access the parameters for each time-step. How can I go about doing that?

**Comment by btaba** [comment]
Hi @blancamiller , thanks for using brax! Could you please describe a bit more about your use-case? What do you mean by parameter values for each time-step? Parameters don't change during a rollout, they get updated over a batch of episodes.
